apps/server/tsconfig.json

```diff
@@ -3,6 +3,10 @@
   "include": ["."],
   "exclude": ["build", "node_modules"],
   "compilerOptions": {
-    "outDir": "./build"
+    "outDir": "./build",
+    "baseUrl": ".",
+    "paths": {
+      "@/*": ["./*"]
+    }
```

Comment:
A QUESTION: is this automatically generated when you installed radix?

Reply:
yes